build.sbt

```
@@ -100,8 +100,7 @@ val sharedJVMSettings: List[Def.Setting[_]] = List(
 ) ++ mimaEnable
 val sharedJSSettings: List[Def.Setting[_]] = List(
   skipIdeaSettings,
-  crossScalaVersions := allScalaVersions.filterNot(_.startsWith("0.")),
-  scalaJSLinkerConfig ~= (_.withModuleKind(ModuleKind.CommonJSModule))
+  crossScalaVersions := allScalaVersions.filterNot(_.startsWith("0."))
```

```suggestion
  crossScalaVersions := allScalaVersions
```
just as a complete side note, I don't think we need this anymore

---

👍 I had a draft comment saying the same thing